doc/examples.rst

```diff
@@ -152,3 +152,123 @@ Instantiated ``init.lua`` content:
 
     require("fiber").sleep(1)
 
+Packing environments
+----------------------------------
+
+For example, we want to pack a single application. Here is the content of the sample application::
+      single_environment/
+      ├── tarantool.yaml
+      └── init.lua
+
+``tarantool.yaml``:
+
+.. code-block:: yaml
+
+    tt:
+        app:
+
+For packing it into tarball, call:
+
+.. code-block:: bash
+
+   $ tt pack tgz
+      • Apps to pack: single_environment
+      • Generating new tarantool.yaml for the new package
+      • The passed bundle packed into temporary directory: /var/folders/T/tt_pack1901831662
+      • Creating tarball.
+      • Bundle is packed successfully to /Users/dev/tt_demo/single_environment/single_environment_0.1.0.0.tar.gz.
+
+The result directory structure::
+
+      unpacked_dir/
+      ├── tarantool.yaml
+      ├── single_environment
+      │   └── init.lua
+      ├── env
+      │   ├── bin
+      │   └── modules
+      ├── instances_enabled
+      │   └── single_environment -> ../single_environment
+      └── var
+          ├── lib
+          ├── log
+          └── run
+
+Example of packing a multi-app environment. The source tree::
+
+     bundle/
+     ├── tarantool.yaml
+     ├── env
+     │   ├── bin
+     │   │   ├── tt
+     │   │   └── tarantool
+     │   └── modules
+     ├── myapp
+     │   ├── Dockerfile.build.cartridge
+     │   ├── Dockerfile.cartridge
+     │   ├── README.md
+     │   ├── app
+     │   ├── bin
+     │   ├── deps.sh
+     │   ├── failover.yml
+     │   ├── init.lua
+     │   ├── instances.yml
+     │   ├── myapp-scm-1.rockspec
+     │   ├── pack-cache-config.yml
+     │   ├── package-deps.txt
+     │   ├── replicasets.yml
+     │   ├── stateboard.init.lua
+     │   ├── systemd-unit-params.yml
+     │   ├── tarantool.yaml
+     │   ├── test
+     │   └── tmp
+     ├── myapp2
+     │   ├── app.lua
+     │   ├── data
+     │   ├── etc
+     │   ├── myapp2
+     │   ├── queue
+     │   ├── queue1.lua
+     │   └── queue2.lua
+     ├── myapp3.lua
+     ├── app4.lua
+     ├── instances_enabled
+     │   ├── app1 -> ../myapp
+     │   ├── app2 -> ../myapp2
+     │   ├── app3.lua -> ../myapp3.lua
+     │   ├── app4.lua -> /Users/dev/tt_demo/bundle1/app4.lua
+     │   └── app5.lua -> ../myapp3.lua
+     └── var
+         ├── lib
+         ├── log
+         └── run
+
+``tarantool.yaml``:
+
+.. code-block:: yaml
+
+    tt:
+        app:
```

Comment:
Suppose, this will not work. Because instances_enabled is . by default, which is single-app in current dir. But we have multi-app bundle here.

Reply:
Fixed, thank you.